src/options.toml

```diff
@@ -34,6 +34,43 @@
 #All of these options are required
 #If they are not provided the program will exit early
 [BCs]
+    # Tells the program that we'll have changing BCs
+    # If left out of the file this defaults to a value of false
+    changing_ess_bcs = false
+    # The below gives an example of how to make use of changing essential BCs
+    # where we have the sample cyclically loaded.
+    # It would go in tandem with using the auto time stepper with a dt = 0.1
```

Comment:
The "auto" time stepping means fixed dt? Maybe `fixed` is then a better name? I think of auto time stepping as adjusting the time step size as the simulation progresses. Which can be a really useful feature ...

Reply:
It does mean fixed dt. I can change the name for the case of a future auto time stepping scheme. However, I'll want to do a deprecation time period and give people time to swap over to the new name. A future auto-time stepping feature will also probably require me to modify the changing BC feature to allow the user to specify a time to change the BCs at.